components/omega/OmegaBuild.cmake

```diff
@@ -0,0 +1,164 @@
+###########################
+# Internal variables      #
+###########################
+
+set(OMEGA_PROJECT_NAME            "OmegaOceanModel")
+set(OMEGA_EXE_NAME                "omega.exe")
+set(OMEGA_LIB_NAME                "OmegaLib")
```

Comment:
Checking: Do we want CamelCase for library name or just lowercase (like in exe)?

Reply:
Yes, we're following llvm standards that use CamelCase with initial capital letter for classes and that follows to most filenames and it makes sense to propagate up to library name too.

Reply:
I agree to use CamelCase for lib names. The executable name does not use CamelCase because I thought it is convention to use lowercase for exe name, but if preferred to use CamelCase, I am good to change.